### ISSUE TITLE
Idea: getting feature flags from dynaconf and pydantic-settings

### ISSUE THREAD

**Issue by imrehg** [Pytlicek/fastapi-featureflags#14]
Besides loading from JSON and dicts, it could be useful to have an easy way to load a set of feature flags from 
- [dynaconf](https://www.dynaconf.com/) which is widely used for configuration, and uses a [dict-like](https://www.dynaconf.com/#reading-settings-variables) interface
- [pydantic-settings](https://docs.pydantic.dev/latest/usage/pydantic_settings/) that FastAPI seems to [endorse](https://fastapi.tiangolo.com/advanced/settings/#pydantic-settings) as well, though I see it less in practice

This could expand the usage of this module, and would make it easier to integrate it.

**Comment by Pytlicek** [comment]
Good idea. I will look at this next week or so. Perhaps an idea for the next Hacktober fest :)

**Comment by UnoYakshi** [comment]
Any updates so far?
